ci/upload-artifacts-template.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Azure template for archiving pipeline step outputs and uploading them.
+#
+# This template will archive all of $BIN_DIR, and upload it for use by
+# downstream jobs using download_artifacts_template.yml.
+#
+# This template expects that a variable $BUILD_ROOT is set. See 
+# util/build_consts.sh for more information.
```

Comment:
Can you check for that? Otherwise the build_consts.sh will use a default value which gives unexpected results. (A simple test should be sufficient, no need for a fancy error message.)

Reply:
Addressed with line 16 below.